anomalib/deploy/optimize.py

```diff
@@ -24,7 +24,7 @@
 import torch
 from torch import Tensor
 
-from anomalib.core.model.anomaly_module import AnomalyModule
+from anomalib.models.components.anomaly_models import AnomalyModule
```

Comment:
I also feel like since `AnomalyModule` is a base class that is used almost everywhere, access to this class should be much simpler. So, rather than `from anomalib.models.components.anomaly_models import AnomalyModule`, I should be able to import it via `from anomalib import AnomalyModule`. Just like LightningModule in Pytorch Lightning

Reply:
This is an additional comment regardless of the location of `AnomalyModule`. Whether it's in `anomalib.core` or `anomalib.models.components.anomaly_models`, we should provide a relative import.